cfc-testing/scripts/enhanced-synth-test.sh

```
@@ -151,12 +177,31 @@ run_synthesis() {
     echo "  🔧 Synthesizing..."
     cd "$BASE_DIR"
 
+    # Temporarily override cdk.json to use CloudForgeCommunitySample (non-interactive)
+    local original_cdk_json="$BASE_DIR/cdk.json"
+    local backup_cdk_json="$BASE_DIR/cdk.json.backup"
+    cp "$original_cdk_json" "$backup_cdk_json"
+
+    # Read the deployment context and inject it into cdk.json
+    local cfc_context=$(cat "$BASE_DIR/deployment-context.json")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)